setup.py

```diff
@@ -50,6 +50,11 @@
     else "deepsparse-nightly"
 )
 
+if is_enterprise:
+    license = "Apache"
+else:
+    license = "Neural Magic DeepSparse Community License, Apache"
+
 if is_enterprise:
```

Comment:
nit: maybe merge with this if statement?

Reply:
yeah and there's also the package name above that also checks this flag. i think in interest of not touching code that already works i won't merge the ifs. `setup.py` sketches me out too much haha